sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusRetryPolicy.cs

```diff
@@ -81,6 +81,16 @@ public abstract class ServiceBusRetryPolicy
             Exception lastException,
             int attemptCount);
 
+        /// <summary>
+        ///   Calculates the number of retry attempts remaining.
+        /// </summary>
+        ///
+        /// <param name="currentTryCount">The current try count.</param>
+        ///
+        /// <returns>The number of retry attempts remaining.</returns>
+        ///
+        protected abstract int CalculateRemainingRetries(int currentTryCount);
```

Comment:
Move this above the internal members to preserve ordering by least visibility to most.